README.md

```diff
@@ -7,60 +7,102 @@ BTCPayServer is an open-source, self-hosted payment processor for Bitcoin and ot
 If you have any issues with BTCPay, please file a [Github issue](https://github.com/btcpayserver/btcpayserver/issues).
 If you have more general questions, please come on [Slack](http://slack.btcpayserver.org/) or check our [Wiki Page](https://nbitstack.com/c/btcpayserver) for common issues.
 
-This solution is for you if:
-
-* You want to be in control of your own funds
-* Compliance teams of traditional payment processors rejected your application
-* You want lower fees (we support SegWit)
-* You want a way to support payments in altcoins
-* You want to accept payments on the Lightning Network
-* You want universally compatible invoices (we support the Bitpay API)
-
 ![BTCPayInfographic](img/BTCPAYINFOGRAPHIC.png)
 
+## Features
+* Direct, peer-to-peer Bitcoin and altcoin payments
+* No transaction fees (other than those for the crypto networks)
+* No processing fees
+* No middleman
+* No KYC
+* User has complete control over private keys
+* Enhanced privacy
+* Enhanced security
+* Self-hosted
+* SegWit support
+* Lightning Network support (LND and c-lightning)
+* Altcoin support
+* Full compatibility with BitPay API (easy migration)
+* Process payments for others
+* Easy-embeddable Payment buttons
+* Point of sale app
+
+## How it works
+### In a nutshell
+In layman words, BTCPay Server is a self-hosted and automated invoicing system. When checking out, the customer is presented with an invoice. Each time, a different address is tied to a particular invoice, so there is no address re-use. BTCPay follows the status of the invoice through the blockchain and informs a merchant when he can fulfill the order safely.
+### How is it different
+BTCPay is a code, not a company. There is no third-party between a merchant and a customer. Merchant is always in full control of the funds. There are no processing or subscription fees. BTCPay Server is free to use and completely open-source, so developers or security auditors can always inspect the quality of the code.
+### How it keeps funds secure
+Payments via BTCPay are direct, peer to peer. The merchant receives the coins directly to the wallet, with no intermediate. Securing your funds comes down to securing your private keys. There is an internal wallet inside BTCPay that merchants can use to spend the funds. The funds from that wallet can only be spent, if a transaction is signed with a compatible hardware wallet.
```

Comment:
```suggestion
Payments via BTCPay are direct, peer to peer. The merchant receives the coins directly to their wallet, with no intermediate. Securing funds comes down to the merchant securing their own private keys, which is standard for any cryptocurrency hardware wallet. There is an internal wallet inside BTCPay that merchants can use to view funds received by paid invoices. The funds from that wallet can only be spent, if a transaction is signed with the merchant's compatible hardware wallet. This allows merchants to have complete control of funds received.
```

Reply:
I think the internal wallet is probably confusing to new users. I tried to clarify the difference.